docs/quick_start_guide.md

```diff
@@ -54,7 +54,7 @@ That's it! Now you're ready to [build your first neural network](create_your_fir
    * Add the following to the `build.gradle` file in the application module:
    ```kotlin
    dependencies {
-       implementation ("org.jetbrains.kotlinx:kotlin-deeplearning-onnx:[KOTLIN-DL-VERSION]")
+       implementation ("'org.jetbrains.kotlinx:kotlin-deeplearning-onnx-android:[KOTLIN-DL-VERSION]")
```

Comment:
With multiplatform magic It should work also without explicitly specifying the "android" suffix.